plugins/pencilblue/controllers/actions/salesforce/change_email.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+const passport = require('koa-passport');
+const request = require('request-promise');
+
+module.exports = function SalesforceChangeEmailControllerModule(pb) {
+
+    //dependencies
+    /**
+     * Changes an user email
+     * @class SalesforceChangeEmailController
+     * @constructor
+     */
+    class SalesforceChangeEmailController extends pb.BaseController {
+        render(cb) {
+            this.sanitizeObject(this.body);
+            this.changeEmail(cb);
+        }
+
+        async changeEmail(cb) {
+            try {
+                const user = this.ctx.session.authentication.user;
+                if (user && user.salesforce) {
+                    const url = `${user.salesforce.authorize.instance_url}/services/data/v44.0/sobjects/User/${user.salesforce.profile.id}`;
+                    const response = await request({
+                        method: 'PATCH',
+                        uri: url,
+                        body: {
+                            Email: this.body.newEmail
+                        },
+                        headers: {
+                            Authorization: `Bearer ${user.salesforce.authorize.access_token}`
+                        },
+                        json: true
+                    });
+                } else {
+                    pb.log.error('User is not logged in via salesforce: ', user);
```

Comment:
Is it possible to change this to a warn level log?  The issue with error is this is a "client" error.  Someone who is not supposed to hit this route hits it.  That way someone could spike our error rate by hitting this route over and over and that is not ideal.